### PR TITLE
update link

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/2622

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/deploy-file-collaboration.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ This is part of the broader Microsoft 365 collaboration story:
 
 - [Meetings and conferencing in Microsoft Teams](/microsoftteams/deploy-meetings-microsoft-teams-landing-page)
 
-We recommend that you download the [Microsoft Teams and related productivity services in Microsoft 365 for IT architects](/microsoft-365/solutions/cloud-architecture-models) poster and refer to it while you read this article. This poster provides detailed illustrations of how the collaboration services in Microsoft 365 relate to each other and interact.
+We recommend that you download the [Microsoft Teams and related productivity services in Microsoft 365 for IT architects](https://docs.microsoft.com/en-us/microsoft-365/solutions/productivity-illustrations?view=o365-worldwide) poster and refer to it while you read this article. This poster provides detailed illustrations of how the collaboration services in Microsoft 365 relate to each other and interact.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/deploy-file-collaboration.md:42]:
```suggestion
We recommend that you download the [Microsoft Teams and related productivity services in Microsoft 365 for IT architects](https://docs.microsoft.com/microsoft-365/solutions/productivity-illustrations?view=o365-worldwide) poster and refer to it while you read this article. This poster provides detailed illustrations of how the collaboration services in Microsoft 365 relate to each other and interact.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please remove the locale from the link.

**Comment by scanum** [comment]
@VLG17 - Please make the requested changes

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**